packages/ttypescript/src/PluginCreator.ts

```diff
@@ -124,7 +124,24 @@ export class PluginCreator {
         this.validateConfigs(configs);
     }
 
-    createTransformers(params: { program: ts.Program } | { ls: ts.LanguageService }) {
+    mergeTransformers(into: ts.CustomTransformers, source: ts.CustomTransformers | TransformerBasePlugin) {
+        const slice = <T>(input: T | T[]) => Array.isArray(input) ? input.slice() : [input]
+        if (source.before) {
+            if (into.before) into.before.push(...slice(source.before));
+            else into.before = slice(source.before);
+        }
```

Comment:
So, into.before,after,afterDeclaration would be always array

Reply:
`ts.CustomTransformers` defines then as always array